swagger_parser_pages/lib/content/generator_content.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: prefer_final_fields
```

Comment:
Why you use this ignore?

Reply:
https://github.com/Carapacik/swagger_parser/pull/97/commits/8a990f7f235fd3317ec9711d621affa667449309